packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges.tsx

```diff
@@ -36,6 +38,17 @@ export const AccountLevelBadges: React.FunctionComponent<
   }
   const hasBadges = Object.values(badgeStatuses).some(value => value)
 
+  const {
+    showButton: showRevokeCertificationButton,
+    buttonProps: revokeCertificationButtonProps,
+    confirmationDialogProps: revokeCertificationConfirmationDialogProps,
+  } = useRevokeCertificationUI({
+    userId,
+    buttonProps: {
+      variant: 'outlined',
+    },
+  })
```

Comment:
Use new hook that encapsulates UI business logic for revoking certification.

I took this approach because `AccountLevelBadge` was pure UI logic (no business logic), so I couldn't add it there. This component handles all of the badges, so it would be messy to inline the logic here.

Using hooks to generate UI props is common in some other libraries (I have seen MUI do it a lot), but we haven't done it much, so I wanted to call it out.